components/org.wso2.carbon.identity.application.authenticator.push/src/main/resources/artifacts/wait.jsp

```diff
@@ -100,13 +101,14 @@
 
             const urlParams = new URLSearchParams(window.location.search);
             sessionDataKey = urlParams.get('sessionDataKey');
-            $.ajax(biometricEndpointWithQueryParams + sessionDataKey, {
+            challenge = urlParams.get('challenge');
+            $.ajax(pushEndpointWithQueryParams + sessionDataKey, {
                 async: false,
                 cache : false,
                 method: GET,
                 // todo: check whether there are any problems when same get req is sent over n over again, will there be cache issues? any solutions?-Future Improvement-Include in DOCs
                 success: function (res) {
-                    handleStatusResponse(res);
```

Comment:
lets remove the todo comment above and open a git issue if that is still an issue

Reply:
Tracked by #17